tests/fixtures/InMemoryStore.ts

```diff
@@ -0,0 +1,46 @@
+import { Pluto } from "../../src/pluto/Pluto";
+
+/**
+ * WARNING: Do not  use this Pluto Store implementation, its for test purposes only.
+ * Persistence is inMemory and totally unprotected.
+ */
+export class InMemoryStore implements Pluto.Store {
```

Comment:
User's that will be implementing this inMemory stores will need to still have their own models.
The problem i think i see is, we are defining the models on our side, the selector is a partial of T, but on the user's side it will have its own models. Can we clarify that part please??

For the query stuff we probably want to grab what i developed for pluto en not reinvent that part, it will be too much work.

Reply:
I'm not sure I understand, the models we have defined are supposed to be the models used,  why do you think they would need to have their own?